version.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
```

Comment:
Please also rename the file to `versions.tf`, so it matches the other modules. Thx!

Reply:
Done!